homeassistant/components/rabbitair/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "domain": "rabbitair",
+  "name": "Rabbit Air",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/rabbitair",
+  "requirements": [
+    "python-rabbitair==0.0.6"
+  ],
+  "ssdp": [],
+  "zeroconf": [
```

Comment:
I don't see the configuration flow step for this implemented?

Reply:
Added a step for zeroconf.

I have a question about this. I tried to update `CONFIG_HOST` with the current IP, using `_abort_if_unique_id_configured(updates={CONFIG_HOST: discovery_info.host})` on zeroconf events. But if such a change took place at the moment of starting `hass`, an error would appear in the logs:
```
ERROR (MainThread) [homeassistant.config_entries] Error unloading entry RabbitAir-XXXXXXXXXXXX for fan
Traceback (most recent call last):
  File "core/homeassistant/config_entries.py", line 467, in async_unload
    result = await component.async_unload_entry(hass, self)
  File "core/homeassistant/components/fan/__init__.py", line 195, in async_unload_entry
    return await component.async_unload_entry(entry)
  File "core/homeassistant/helpers/entity_component.py", line 172, in async_unload_entry
    raise ValueError("Config entry was never loaded!")
ValueError: Config entry was never loaded!
```
However, it did not affect anything, after that everything worked.

If the configuration was updated from the same handler when `hass` was already running, no errors occurred.

What could be the reason? Maybe I'm doing something wrong?

I have now removed this update as it's not really needed that much. The rabbitair library can resolve the mDNS names itself if a zeroconf object was provided during initialization.